ui/src/app/base/components/FormikFormContent/FormikFormContent.tsx

```diff
@@ -87,7 +89,7 @@ const FormikFormContent = <V, E = null>({
   savedRedirect,
   saving,
   submitDisabled,
-  "aria-label": ariaLabel,
+  "aria-label": ariaLabel = "form",
```

Comment:
```suggestion
  "aria-label": ariaLabel,
```

In this case it's better to have no label rather than a generic one.

Assistive tech recognises HTML semantics, no need to describe that it's a form. All this will do is make the screen reader read out "form" twice - screen readers typically announce a role followed by a label (in this case "form, form").

Reply:
Ah thanks this made me revisit why this was necessary. To have `role="form"` you need to set the label to anything, not to `form`. I've set a proper label on the EditVLAN form now.